modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/pagination/Page.java

```diff
@@ -174,8 +179,8 @@ private Page(
 	}
 
 	private final Map<String, Map<String, String>> _actions;
+	private Map<String, List<Facet>> _facets = new HashMap<>();
```

Comment:
Not sure if I totally understand the Map usage. It is to group the facets that refers to values of the same field?

Reply:
I want to rename facet, because it's wrongly used here, values? buckets? facetValues?

What we want to show is a Map of a list of term-value.

```
"facets": {
    "keywords": [
      {
        "numberOfOccurrences": 2,
        "term": "hola"
      },
      {
        "numberOfOccurrences": 1,
        "term": "adios"
      }
    ],
    "userName": [
      {
        "numberOfOccurrences": 2,
        "term": "user1 user1"
      },
      {
        "numberOfOccurrences": 1,
        "term": "test test"
      }
    ]
  },
```

Reply:
facetedSearch, facetResult or facetValues? I'd prefer something related with facet to avoid adding a new concept that the user may not directly link with facets